lib/aquilon/worker/templates/cluster.py

```diff
@@ -188,7 +188,15 @@ def get_key(self, exclusive=True):
         return CompileKey.merge(keylist)
 
     def body(self, lines):
+        # Allow settings such as loadpath to be modified by the archetype before anything else happens
+        # Included only if template_library_configured option is true
+        # This template is optional: included only if it exists
+        if self.config.getboolean("panc", "object_preface_template"):
+            pan_include(lines, "archetype/loadpath")
```

Comment:
I think you missed: lines.append("") after pan_include

Reply:
I thought you asked me to delete it in a previous comment.. I'll readd it!